src/main/java/com/accelad/acctive/sim/kernel/core/evaluator/diff/BetweenStrictlyLessThanMaxFunction.java

```diff
@@ -30,7 +30,7 @@ public X getValue() {
         double a = firstBound.getValue().getReal();
         double b = secondBound.getValue().getReal();
 
-        if (v >= Math.min(a, b) && v <= Math.max(a, b)) {
+        if (v >= Math.min(a, b) && v < Math.max(a, b)) {
             return DFFactory.get(1d);
```

Comment:
It's probably not in the scope of that PR, but why does this method return 0/1 and not a boolean ?